lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -1942,7 +1941,7 @@ reapplyFunctionConversion(SILValue newFunc, SILValue oldFunc,
     auto innerNewFunc = reapplyFunctionConversion(newFunc, oldFunc,
                                                   cfi->getOperand(), builder,
                                                   loc, newFuncGenSig);
-    // Match a conversion from escaping to `@noescape`
+    // Match conversion from escaping to `@noescape`.
```

Comment:
I think this matching logic can be dropped because of #26242.

Reply:
Done in https://github.com/apple/swift/pull/26292/commits/6e170f2228ce1f0e13b3e1d5dc988d4f8a82580f, confirmed tests pass.